sql/changes/1.6/country_date_format.sql

```diff
@@ -0,0 +1,267 @@
+ALTER TABLE country
+  ADD date_format TEXT;

```

Comment:
1- Dojo must know at start of application which languages will be used
2- Even if Dojo can display, convert, calculate on dates, it doesn't seem to have a way to show a date format by country of locale.